docs/api.md

```diff
@@ -219,6 +219,7 @@
   * [elementHandle.getProperties()](#elementhandlegetproperties)
   * [elementHandle.getProperty(propertyName)](#elementhandlegetpropertypropertyname)
   * [elementHandle.hover()](#elementhandlehover)
+  * [elementHandle.isDisplayed()](#elementhandleisdisplayed)
```

Comment:
nit: I like isVisible over isDisplayed, because we already use the word visible in waitForSelector.